src/api.rs

```diff
@@ -386,6 +399,9 @@ impl Default for ColorPrimaries {
     }
 }
 
+/// Contains different types of Transfer Characteristics, they define the optoelectronic tranfer charecteristic of the source picture aka Gamma.
+/// - 8 Bit field is sufficient for representation of the TransferCharacteristics.
```

Comment:
how is this information useful, considering that it's true for every other color field?